### PR TITLE
squashed AstropyDeprecationWarning in vso.attrs.Wavelength

### LINKED ISSUE #3058: vso.attrs.Wavelength raises AstropyDeprecationWarning
Can't work out how to fix this at the moment, so leaving a bug as a placeholder:
```python
from sunpy.net import vso
import astropy.units as u
vso.attrs.Wavelength(12 * u.AA, 16 * u.AA)
```
results in 
```
WARNING: AstropyDeprecationWarning: The truth value of a Quantity is ambiguous. In the future this will raise a ValueError. [astropy.units.quantity]
```

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes #3058

### CODE REVIEW

File: sunpy/net/vso/attrs.py [hunk 1]
```diff
@@ -18,7 +18,6 @@
 import collections
 
 import astropy.units as u
-from astropy.time import Time
```
Comment (nabobalis) [inline sunpy/net/vso/attrs.py]: So this change I'm unsure about. Within the `Time` attr, the method `pad` returns a `Time`but I assume this must be a new `Time` attr and not astropy.time.Time instance. If that is the case, then we don't need this import.

**Comment by nabobalis** [comment]
Online build failure is database doc example complaining about the database being locked.

File: sunpy/net/vso/attrs.py (hunk 1 above)
Comment (Cadair) [reply sunpy/net/vso/attrs.py]: This import would get clobbered by the `Time` attr anyway right? I think you should be good here.

Comment (nabobalis) [reply sunpy/net/vso/attrs.py]: It's what the linter said should happen.

**Pull request merged**